### ISSUE TITLE
Incorporating previous binning attempts

### ISSUE THREAD

**Issue by hrwaik** [EMBL-PKU/BASALT#10]
Hello,

I was wondering if it was possible to include previous binning results into BASALT at the refinement stage? I have multiple previous outputs from other binners not included in BASALT and would like to combine these with the results of BASALT's autobinner stage.

Thanks
Huw

**Comment by EMBL-PKU** [comment]
Yes. BASALT has a way to feed previous binning bins into it. And then you could the refinement module.  To do this, you need to manually use the script "Data_fedding_02282021.py" to feed the data to BASALT. And then, you may use the S5_outlier_remover_DL script to carry out the refinement process. Could you please send me an email to yuke.sz@pku.edu.cn? I will prepare a detailed instruction for doing this. I think I could make an automatic pipeline to accomplish this goal.

**Comment by EMBL-PKU** [comment]
Sorry for the late response :(

**Comment by camillaln** [comment]
Hi - I am also interested  in doing this. I have two sets of MAGs produced by BASALT that I want to compare to a previous binning dataset. (I could not run all assemblies together due to memory issues ...)
Camilla
